src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/ui_get_capabilities_response.cc

```diff
@@ -128,6 +128,11 @@ void UIGetCapabilitiesResponse::Run() {
                     [strings::display_capabilities]);
     }
   }
+
```

Comment:
@SNiukalov Please notice name of source branch does not follow our standard agreement: `feature/<name>`